projects/wvr-elements/src/lib/wvr-button/wvr-button.component.spec.ts

```diff
@@ -28,11 +28,6 @@ describe('WvrButtonComponent', () => {
       .toEqual('primary');
   });
 
-  it("should have as btnSize 'large'", () => {
```

Comment:
Is this test no longer applicable?

Reply:
Is btnSize a required attribute now? If not, what is the default now?

Reply:
This test is no longer applicable.